wait/awaitility.go

```diff
@@ -54,10 +60,27 @@ type SingleAwaitility interface {
 }
 
 type SingleAwaitilityImpl struct {
-	T               *testing.T
-	Client          framework.FrameworkClient
-	Ns              string
-	OtherOperatorNs string
+	T                     *testing.T
+	Client                framework.FrameworkClient
+	Ns                    string
+	OtherOperatorNs       string
+	OperatorRetryInterval time.Duration
+	OperatorTimeout       time.Duration
```

Comment:
why do we have `OperatorRetryInterval` and `OperatorTimeout` when they are not used anywhere?

Reply:
Good question :)
Let me double check and clean them up.

Reply:
Done in https://github.com/codeready-toolchain/toolchain-e2e/pull/46/commits/44cc75736a74e317729701874ce1ed520d54208a